lib/exception_notifier/webhook_notifier.rb

```diff
@@ -1,5 +1,6 @@
 require 'action_dispatch'
 require 'active_support/core_ext/time'
+require 'action_dispatch/http/request' # for filtered_env
```

Comment:
what if the gem actionpack is not included? i wonder if it would be better to check `defined?(ActionDispatch::Request)`